.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: ❓ Questions/Help
+about: If you have questions, please check Aragon Hack, Aragon Forum or Aragon Chat
+---
+
+## ❓ Questions and Help
+
+### Please note that this issue tracker is not a help form and this issue will be closed.
+
+Please contact us instead. We have a few channels:
+
+- [Aragon Hack](https://hack.aragon.org/docs)
+- [Aragon Forum](https://forum.aragon.org)
+- [Aragon Chat](https://aragon.chat)
```

Comment:
Lets use the spectrum link instead: https://spectrum.chat/aragon

Reply:
Fixed with https://github.com/aragon/buidler-aragon/pull/132/commits/36cdee616e35e5ce507b29c6a5511e6807737474